extensions/redis-client/runtime/src/main/java/io/quarkus/redis/runtime/datasource/ReactiveTransactionalCountMinCommandsImpl.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.redis.runtime.datasource;
+
+import java.util.List;
+import java.util.Map;
+
+import io.quarkus.redis.datasource.countmin.*;
```

Comment:
nit: remove usage of wildcard import?

Reply:
Fixed (and a few others). This is due to the generator I used.... Need to fix it :-D

Reply:
Thanks